acceptance/ui/features/services.feature

```diff
@@ -29,4 +29,183 @@ Feature: Service Management
       And I should see "Public Endpoints"
       And VHost "table://services/childServiceWithVHost/vhostName" should exist
       And Public Port "table://services/childServiceWithVHost/publicPortNumber" should exist
-      And I should see "table://services/childServiceWithVHost/configFile"
\ No newline at end of file
+      And I should see "table://services/childServiceWithVHost/configFile"
+
+  @services_endpoint
+  Scenario: Public Endpoint dialog - check all labels
```

Comment:
Please add a test case that verifies that dialog is populated with all of the correct values for each field